Makefile

```diff
@@ -1,10 +1,26 @@
-PROMBENCH_CMD        = ./prombench
-DOCKER_TAG = docker.io/prombench/prombench:2.0.0
+PROMBENCH_CMD := ./prombench
+DOCKER_REPO := prombench
+DOCKER_IMAGE_NAME := prombench
+DOCKER_IMAGE_TAG := 2.0.0
 
+include Makefile.common
+
+# This is to prevent go get promu, staticcheck & govendor from updating go.mod 
+export GO111MODULE = off
+
+.PHONY: build
+build: promu
+	@echo ">> building binaries"
+	@go version | grep go1.11 || exit  "Requires golang 1.11 with support for modules!"
+	@GO111MODULE=on $(PROMU) build
```

Comment:
Same here, it would be better to have `Makefile.common` being able to deal with Go modules.